tensorflow/core/platform/gif.h

```diff
@@ -20,7 +20,7 @@ limitations under the License.
 
 #if defined(PLATFORM_GOOGLE)
 #include "tensorflow/core/platform/google/build_config/gif.h"
-#elif (defined(PLATFORM_POSIX) && !defined(IS_MOBILE_PLATFORM)) || defined(PLATFORM_WINDOWS)
```

Comment:
I'm a little concerned this may trigger problems from Android builds that don't link against the image libs, but that should show up in the build tests if so, so no need to change this yet.